src/shared/links/PrimaryButtonLink.js

```diff
@@ -0,0 +1,21 @@
+import { Link } from "react-router-dom";
```

Comment:
User Bootstrap btn-primary class and modify bootstrap btn scss snipet in _custome.bootstrap.scss on app.js

Reply:
Done as requested